runtime/trampolines-varargs.m

```diff
@@ -132,7 +128,7 @@
 	case _C_ULNG:
 	case _C_LNG_LNG:
 	case _C_ULNG_LNG:
-		state->longlong_ret = *(uint64_t *) mono_object_unbox (value);
```

Comment:
@rolfbjarne the answer to your question is this case here. We could change it to long long, I did not want to loose precision and supposed you did the explicit cast for a reason.

Reply:
The explicit cast is there because `mono_object_unbox` returns a void*.

This can just as easily be:

```suggestion
		state->longlong_ret = *(int64_t *) mono_object_unbox (value);
```

Reply:
Reverted and made the commit with the suggestion.